frontend_app_folder_advanced/src/container/PopupCreateFolder.jsx

```diff
@@ -87,7 +88,7 @@ class PopupCreateFolder extends React.Component {
   handleChangeNewFolderName = e => this.setState({newFolderName: e.target.value})
 
   handleClose = () => GLOBAL_dispatchEvent({
-    type: 'hide_popupCreateContent', // handled by tracim_front:dist/index.html
+    type: CUSTOM_EVENT.HIDE_POPUP_CREATE_CONTENT, // handled by tracim_front:dist/index.html
```

Comment:
now we have a variable, it is easier to find where are the usages of a given custom event.
So you can remove this kind of comments